apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts

```diff
@@ -110,7 +109,7 @@ export function getSwapButtonState(input: SwapButtonStateParams): SwapButtonStat
     return SwapButtonState.ReadonlyGnosisSafeUser
   }
 
-  if (isQuoteExpired(quote?.fee?.expirationDate) && !swapBlankState && hasInput) {
+  if (isQuoteExpired(quote?.fee?.expirationDate) && trade && !input.inputError) {
```

Comment:
This is already included in the release... shouldn't be showing up here 🤔

Reply:
I think this is because of changing target branch

Reply:
Fixed